test/skimNtuple2018.cpp

```diff
@@ -1649,6 +1655,51 @@ int main (int argc, char** argv)
             }
         }
 
+      // For ttHToNonBB only: loop on gen particles to find the Higgs
+      // if the Higgs decays to TauTau --> throw away the event!
+      if (isttHToNonBB)
+      {
+        // Loop on gen to find Higgs
+        int idx1 = -1;
+        for (unsigned int igen = 0; igen < theBigTree.genpart_px->size(); igen++)
+        {
+          bool isLast = CheckBit (theBigTree.genpart_flags->at(igen), 13) ; // 13 = isLastCopy
```

Comment:
Does it need testing or is it working fine like this?
(by working fine I mean that you always find exactely one H in all ttH events)

Reply:
I think it's fine like this.
I tested it on 300k events and i've never seen an error like "more than 1 H identified".
As a further check, I also tested the code on a `ttHToTauTau` file: in this case all events are H->TauTau decays and all events are correctly rejected.